test/exq_test.exs

```diff
@@ -117,6 +117,30 @@ defmodule ExqTest do
     assert_received {:worked, 2}
     stop_process(sup)
   end
+  
+  test "register queue and run job" do
+    Process.register(self, :exqtest)
+    {:ok, sup} = Exq.start_link([name: :exq_t, port: 6555, namespace: "test", queues: ["q1"]])
+    {:ok, _} = Exq.subscribe(:exq_t, "q2", 10)
+    {:ok, _} = Exq.enqueue(:exq_t, "q1", ExqTest.PerformArgWorker, [1])
+    {:ok, _} = Exq.enqueue(:exq_t, "q2", ExqTest.PerformArgWorker, [2])
+    wait_long
+    assert_received {:worked, 1}
+    assert_received {:worked, 2}
+    stop_process(sup)
+  end
+  
+  test "unregister queue and run job" do
+    Process.register(self, :exqtest)
+    {:ok, sup} = Exq.start_link([name: :exq_t, port: 6555, namespace: "test", queues: ["q1","to_remove"]])
+    {:ok, _} = Exq.unsubscribe(:exq_t, "to_remove")
```

Comment:
Investigated a bit more last night and saw that running the new tests before the all the other tests in exq_tests.exs results in a success, but putting it after "start using registered name" will cause the fail. The plot thickens!